frontend/src/components/osh/OSHScansTable.tsx

```diff
@@ -57,19 +58,32 @@ const OSHScansTable = () => {
     <Th key={columnNames.newFindings} width={15}>
       {columnNames.newFindings}
     </Th>,
-    <Th key={columnNames.timeProcessed} width={20}>
+    <Th key={columnNames.timeProcessed} width={10}>
       {columnNames.timeProcessed}
     </Th>,
-    <Th key={columnNames.scan} width={20}>
+    <Th key={columnNames.scan} width={15}>
       {columnNames.scan}
     </Th>,
+    <Th key={columnNames.allFindings} width={15}>
+      {columnNames.allFindings}
+    </Th>,
   ];
 
   // If backend API is down
   if (isError) {
     return <ErrorConnection />;
   }
 
+  const scans = useMemo(() => {
+    if (!data) return undefined;
+    return data.map((scan) => ({
+      ...scan,
+      scan_results_url: scan.scan_results_url
+        ? scan.scan_results_url.replace(".js?format=raw", ".html")
+        : scan.scan_results_url,
+    }));
+  }, [data]);
```

Comment:
Since we are creating an array we can keep it an array too instead of `undefined`. Likewise the `scan_results_url` can use optional chaining
```suggestion
  const scans = useMemo(() => {
    if (!data) return [];
    return data.map((scan) => ({
      ...scan,
      scan_results_url: scan.scan_results_url?.replace(".js?format=raw", ".html")
    }));
  }, [data]);
```